docs/6.coordination/3.statistics.md

```diff
@@ -0,0 +1,124 @@
+## FIWARE Lab Statistics
+
+### Available Resources
+
+#### Physical resources {#physical-resources .ListParagraph}
+
+According to OpenStack documentation the default overcommit for RAM is
+1.5:1, the default overcommit for CPU is 16:1 and disk space for the
+Virtual Machines should not be overcommitted (please, notice that
+Volumes provisioned by Cinder is not physically where the Virtual
+machines are) -- That is shown in this page of the documentation:
+
+[*https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html*](https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html)
+
+The available physical resources can be easily calculated when the
+hardware is bought and installed. However, we could ask nova about the
+resources (available and used) with a small and simple script like this
+one:
+
+```
+( nova --timeout 15 --insecure list &gt; /dev/null || exit 1\
+  h=$(($(nova --timeout 60 --insecure list --all-tenants | wc -l) - 4))
+  echo hosts $h
+  for a in `nova --timeout 60 --insecure host-list | awk '/compute/ {print $2}'`; do
+    nova --timeout 60 --insecure host-describe $a | awk '
+    /total/ {printf ("cpu %s\nmem %s\ndisk %s\n",$6,$8,$10)}\
+    /used_now/ {printf ("used_cpu %s\nused_mem %s\nused_disk%s\n",$6,$8,$10)}\
+    ' &\
+done ) | awk '{sum[$1] += $2 } ; END {for (a in sum) {print a,sum[a]}}'
+```
+
+The output for this script is something like this:
+```
```

Comment:
docs/6.coordination/3.statistics.md: 33: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]